src/gui_common/FocusGrabber.cs

```diff
@@ -53,7 +53,10 @@ public override void _Ready()
     {
         if (string.IsNullOrWhiteSpace(NodeToGiveFocusTo))
             throw new ArgumentException("Focus grabber must have the node to focus set");
+    }
 
+    public override void _EnterTree()
+    {
         UpdateOverrideFocusStrings();
```

Comment:
There's actually a comment within `UpdateOverrideFocusStrings` about needing to update things while inside the tree. Could you resolve that? This is now a bit of a code cleanness decreasing change as the comment in that method doesn't match reality anymore...

Reply:
I don't fully understand what that comment means, reparenting makes the node exits the tree for a bit so doesn't this change has no relation to what the comment is saying?

Reply:
Reading the comment, I *think* I had a idea in my mind on how the focus grabber could be more performant when it needs to refresh the nodes it is pointing to. If you can't figure out any way to make the code faster for the reparent case, then the comment should be just removed.

Reply:
I guess I'll just remove it.